Source/NNSupportFunctions/arm_nn_mat_mult_nt_t_s8_s32.c

```diff
@@ -54,9 +54,332 @@ arm_cmsis_nn_status arm_nn_mat_mult_nt_t_s8_s32(const int8_t *lhs,
+            int8x16_t v_lhs1 = vldrbq_z_s8(lhs_ptr, rmdr);
+            lhs_ptr += rhs_rows;
+            int8x16_t v_lhs2 = vldrbq_z_s8(lhs_ptr, rmdr);
+            lhs_ptr += rhs_rows;
+            int8x16_t v_lhs3 = vldrbq_z_s8(lhs_ptr, rmdr);
+            lhs_ptr += rhs_rows;
+
+            for (int32_t rhs_cols_idx = rhs_cols; rhs_cols_idx != 0; rhs_cols_idx--)
+            {
+                int32_t *ip_dst = dst_ptr;
+                int8x16_t v_rhs0 = vldrbq_z_s8(rhs_ptr, rmdr);
+
+                int32_t rhs_sum = vaddvq_p_s8(v_rhs0, rmdr);
+                rhs_sum *= lhs_offset;
+
+                *ip_dst += rhs_sum;
+                *ip_dst = vmladavaq_p_s8(*ip_dst, v_lhs0, v_rhs0, rmdr);
+                ip_dst += dst_idx_col_offset;
+
+                *ip_dst += rhs_sum;
+                *ip_dst = vmladavaq_p_s8(*ip_dst, v_lhs1, v_rhs0, rmdr);
+                ip_dst += dst_idx_col_offset;
+
+                *ip_dst += rhs_sum;
+                *ip_dst = vmladavaq_p_s8(*ip_dst, v_lhs2, v_rhs0, rmdr);
+                ip_dst += dst_idx_col_offset;
+
+                *ip_dst += rhs_sum;
+                *ip_dst = vmladavaq_p_s8(*ip_dst, v_lhs3, v_rhs0, rmdr);
+
+                dst_ptr += dst_idx_offset;
+                rhs_ptr += rhs_rows;
+            }
+
+            dst_ptr += 3 * dst_idx_col_offset;
+        }
+        for (; lhs_rows_idx > 0; lhs_rows_idx--)
+        {
+            const int8_t *rhs_ptr = &rhs[0];
+            int8x16_t v_lhs0 = vldrbq_z_s8(lhs_ptr, rmdr);
+
+            for (int32_t rhs_cols_idx = rhs_cols; rhs_cols_idx != 0; rhs_cols_idx--)
+            {
+                int8x16_t v_rhs0 = vldrbq_z_s8(rhs_ptr, rmdr);
+
+                int32_t rhs_sum = vaddvq_p_s8(v_rhs0, rmdr);
+                *dst_ptr += rhs_sum * lhs_offset;
+
+                *dst_ptr = vmladavaq_p_s8(*dst_ptr, v_lhs0, v_rhs0, rmdr);
+
+                dst_ptr += dst_idx_offset;
+                rhs_ptr += rhs_rows;
+            }
+            lhs_ptr += rhs_rows;
+        }
+    }
+
+#elif defined(ARM_MATH_DSP)
+    int16_t lhs_offset_s16 = (int16_t)lhs_offset;
+    const uint32_t lhs_offset_s16x2 = PKHBT(lhs_offset_s16, lhs_offset_s16, 16);
+    for (; rhs_rows_idx >= 8; rhs_rows_idx -= 8)
+    {
+        int32_t *dst_ptr = &dst[0];
+        const int8_t *lhs_ptr = &lhs[0];
+        int32_t lhs_rows_idx = lhs_rows >> 1;
+
+        while (lhs_rows_idx)
+        {
+            const int8_t *rhs_ptr = &rhs[0];
+
+            int32_t lhs000, lhs001, lhs010, lhs011, lhs100, lhs101, lhs110, lhs111;
+            read_pad_and_add(lhs_ptr, &lhs000, &lhs001, lhs_offset_s16x2);
+            read_pad_and_add(&lhs_ptr[4], &lhs010, &lhs011, lhs_offset_s16x2);
+            read_pad_and_add(&lhs_ptr[rhs_rows], &lhs100, &lhs101, lhs_offset_s16x2);
+            read_pad_and_add(&lhs_ptr[rhs_rows + 4], &lhs110, &lhs111, lhs_offset_s16x2);
+
+            for (int32_t rhs_cols_idx = rhs_cols; rhs_cols_idx != 0; rhs_cols_idx--)
```

Comment:
for the loop variables, please use a similar naming format like in other places. 

like i_rhs_cols here..Same applies for all in this file.

Reply:
I'm a bit conflicted on this one, it seems like this naming convention matches other mat_mul files, but not many other files. I think it would be best to leave it for now and potentially revisit files in bulk with a batched style convention review